### DISCUSSION TITLE
Is there any way to customise / localise the back button in a NativeStackNavigator?

### DISCUSSION THREAD

**Discussion by TheLartians** [software-mansion/react-native-screens#783] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/4437447/105483676-7e69f100-5caa-11eb-915d-d467b52d3dae.png)

If a previous title is too long on iOS the label on the back button get's replaced with a "Back" string, regardless of the devices language setting. Is there any way to customise the truncated back title (as with the [standard stack navigator](https://reactnavigation.org/docs/stack-navigator#headertruncatedbacktitle)), or at least use a localised string so the app will show users "Back" in their respective languages?

**Comment by tboba** [comment, marked as answer]
Hi @TheLartians, sorry for the late response 🙏 
> If a previous title is too long on iOS the label on the back button get's replaced with a "Back" string, regardless of the devices language setting.

I believe in react-native-screens 3.26.0 it's right now possible to enter long title in `headerBackTitle` prop, but you can't truncate it yet.
I also think `Back` as a back button title is being controlled by iOS - so it should localise the string itself.

Cheers!
